web/src/components/FilteredConnection.tsx

```diff
@@ -459,7 +459,7 @@ export class FilteredConnection<N, NP = {}, C extends Connection<N> = Connection
             debounceTime(200),
             startWith(this.state.query)
         )
-        const refreshRequests = new Subject<void>()
+        const refreshRequests = new Subject<boolean>()
```

Comment:
You should at least add a docstring to `refreshRequests` explaining what the boolean value represents. I would find it even clearer to make the following change:

```suggestion
        const refreshRequests = new Subject<{ forceRefresh: boolean }>() 
```

At callsites, `refreshRequests.next({ forceRefresh: true })` is much more discoverable than `refreshRequests.next(true)`.

Reply:
Done.